ui/src/components/useMessageDialog.tsx

```diff
@@ -171,13 +191,19 @@ const MessageDialogContainer = () => {
               <Title>{messageDialogParameters.position.job_title}</Title>
             </Info>
           </ContentContainerNested>
-          <TextArea
-            placeholder={messageDialogParameters.position.placeholder}
-            onChange={handleTextAreaChange}
-          />
-          <Button fullWidth primary onClick={() => postMessage()}>
-            Send Message
-          </Button>
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <TextArea
+              placeholder={messageDialogParameters.position.placeholder}
+              name="message"
+              onChange={handleTextAreaChange}
+              ref={textareaRef}
+              error={!!errors.message}
+            />
+            {errors.message && <InputErrorMessage>{errors.message.message}</InputErrorMessage>}
```

Comment:
i like the ternary operator version more 
`errors.message ? <InputErrorMessage>{errors.message.message}</InputErrorMessage> : ''` 
because its more explicit but i understand if this is the norm/code style for the project! Just wanted to bring +1 perspective